config/manager/manager.yaml

```diff
@@ -76,6 +76,8 @@ spec:
               value: quay.io/konveyor/velero-plugin-for-gcp:latest
             - name: RELATED_IMAGE_KUBEVIRT_VELERO_PLUGIN
               value: quay.io/konveyor/kubevirt-velero-plugin:v0.7.0
+            - name: RELATED_IMAGE_HYPERSHIFT_VELERO_PLUGIN
+              value: quay.io/hypershift/hypershift-oadp-plugin:latest
```

Comment:
should we used fixed tag as kubevirt?

Reply:
not for now I think, let's roll this out and once I have all the issues fixed in the rest of the providers I will create the first release